### ISSUE TITLE
tagbox not working with select2 Dynamic option creation

### ISSUE THREAD

**Issue by masciugo** [surveyjs/widgets#33]
it seems from the source that [select2 tags option](https://select2.org/tagging) is ignored 

https://plnkr.co/edit/qj6S8GrZtlatqjk3mv40?p=preview

**Comment by tsv2013** [comment]
You are absolutely right. Currently tag box custom widget doesn't support dynamic tags creation.
Select2 and other 3rd party widgets have a lot of options and support all of them is a challenging task. We have created the [Custom config of 3d party libraries for custom widgets](https://github.com/surveyjs/widgets/issues/32) issue aimed to support options, not implemented by our custom widgets out-of-the-box.

As for dymanic tagging: I've checked link you sent: we are already set tags option to true in the `tagbox` custom widgets. It seems the thing is in the way the options have been created (statically in the html, as in the example) or in the extra `multiple='multiple'` option in our custom widget markup.

I guess your case is not as simple as the Select2 example. Probably then we need to support more complex scenario as custom tokenization, constrains, placement, etc. It can take some time to implement.

You can wait for our implementation of this feature (if we will get another requests from other users) or you can take code of our `tagbox` custom widget and make all nesessary customizations on your side and share your experience with the comunity via pull request into this repo.
